ghost/content.md

```diff
@@ -17,3 +17,11 @@ If you'd like to be able to access the instance from the host without the contai
 	docker run --name some-ghost -p 8080:2368 -d ghost
 
 Then, access it via `http://localhost:8080` or `http://host-ip:8080` in a browser.
+
+You can also point the image to your existing content on your host
+
+	docker run --name some-ghost -v /path/to/ghost/blog:/var/lib/ghost ghost
+
+...or a data container
```

Comment:
How about making this a complete sentence and making it clear that the data container will need to have a `/var/lib/ghost` mount?